src/authentication/azure/AzureIdentityAccessTokenProvider.cs

```diff
@@ -0,0 +1,103 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
```

Comment:
leaving the comment here as I can't on binary files, but maybe we could use a central snk file instead of duplicating it?

Reply:
+1